addons/website_project_issue/views/project_issue_templates.xml

```diff
@@ -0,0 +1,137 @@
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="panel-body">
+                                <div class="mb8">
+                                    <strong>Date:</strong> <span t-field="issue.create_date" t-field-options='{"widget": "date"}'/>
+                                </div>
+                                <div class='row'>
+                                    <div class="col-md-6">
+                                        <strong>Company Contact</strong>
+                                        <div t-if="issue.user_id">
+                                            <address t-field="issue.user_id" t-field-options='{"widget": "contact", "fields": ["name", "email", "phone"], "no_marker": true}'/>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-6">
+                                        <strong>Your Contact</strong>
+                                        <div t-if="issue.partner_id">
+                                            <address t-field="issue.partner_id" t-field-options='{"widget": "contact", "fields": ["name", "email"], "no_marker": true}'/>
+                                        </div>
+                                    </div>
+                                </div>
+                                <span t-field="issue.stage_id.name" class="pull-right label label-danger"/>
+                                <hr/>
+                                <t t-if="not timesheets">
+                                    <strong class="mb8">No timesheets are available for this issue</strong>
+                                </t>
+                                <t t-if="timesheets">
+                                    <strong>Timesheets</strong>
+                                    <table class="table table-hover">
+                                        <thead>
+                                          <tr class="active">
+                                            <th>Date</th>
+                                            <th>Responsible</th>
+                                            <th>Description</th>
+                                          </tr>
+                                        </thead>
+                                        <tr t-foreach="timesheets" t-as="timesheet">
+                                            <td><span t-esc="timesheet[0]"/></td>
+                                            <td><span t-esc="timesheet[1]"/></td>
+                                            <td><span t-esc="timesheet[2]"/></td>
+                                        </tr>
+                                    </table>
+                                </t>
+                            </div>
+                        </div>
+                        <div class="row mt32">
```

Comment:
In a future task, it could be good to factorize this section with https://github.com/odoo/odoo/blob/master/addons/website_quote/views/website_quotation.xml#L339 and put everything in `website_portal`